### DISCUSSION TITLE
Custom css is not working on pdf view

### DISCUSSION THREAD

**Discussion by Pandetthe** [cefsharp/CefSharp#4729] [Q&A]
Hi, I am trying to change css of pdf view page (being more specificly, I want to hide scrollbar) with this code below. This works for normal webpage but not for pdf view.
```csharp
        private void ChromiumBrowser_FrameLoadStart(object? sender, FrameLoadStartEventArgs e)
        {
            string script = @"(function()
{
    var styles='::-webkit-scrollbar { display: none; }';
    var styleSheet = document.createElement(""style"");
    styleSheet.innerText = styles;
    document.head.appendChild(styleSheet);
})();";
            if (e.Frame.IsMain)
                e.Frame.ExecuteJavaScriptAsync(script);
        }
```
Am I doing something wrong or it's not possible?

**Comment by amaitland** [comment]
> Am I doing something wrong or it's not possible?

The PDF viewer is a special case and styling it is not currently supported.
